markdown/chapters/appendices/faq.md

```diff
@@ -334,6 +334,29 @@ Rivulet] and act as a pseudo +2 to your {U} card count for Force of Will
 effects. Especially useful if you need to bring in Force of Negations and
 maximise the ability to use them.
 
+## Why do some lists run Consider without Deep Analysis?
+
+:card[Consider] was first introduced and one of the aspects that made it an
+appealing tool was the fast and low resource piles that could be made with it.
+
+However this was not the key point as to why it was run. It provides a decent
+cantrip effect which can be used to gain advantage through cards like
+:card[Narset, Parter of Veils] whilst also providing a clean way to *dig*
+through two cards in a pile. It also provides a way to create *perfect*
+:card[BS] piles [in key scenarios][meandeck:brainstorm-free-cantrip].
+
+[meandeck:brainstorm-free-cantrip]: /meandeck/brainstorm#brainstorm-pile---brainstorm-and-free-cantrip-available
+
+## When do I board in Shelldock Isle and Emrakul?
+
+When people elect to run :card[SI] and :card[Emrakul, the Aeon's Torn] they are
+generally thinking about fair {U} matchups that do not run :card[Wasteland]. It
+should not be brought in vs. any Delver variant or when Wasteland and
+:card[Teferi, Time Raveler] are prominant in control variants as both of these
+can effect it.
```

Comment:
@Doishy Is it "effect" or "affect"?

Happy with the rest otherwise 👍

Reply:
affect